From a0e65b917b65142f15e5664078b1a27542156dc9 Mon Sep 17 00:00:00 2001
From: Sam Ravnborg <srn@skov.dk>
Date: Fri, 17 Feb 2017 22:18:00 +0100
Subject: [PATCH] Drop DEBUG

No longer needed - so drop it from CONFIG to minimize output to serial
port thus speeding up boot
---
 contrib/board/skov/arm9_cpu/arm9_cpu_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/contrib/board/skov/arm9_cpu/arm9_cpu_defconfig b/contrib/board/skov/arm9_cpu/arm9_cpu_defconfig
index 5cbce37..30d87b4 100644
--- a/contrib/board/skov/arm9_cpu/arm9_cpu_defconfig
+++ b/contrib/board/skov/arm9_cpu/arm9_cpu_defconfig
@@ -3,4 +3,3 @@ CONFIG_RAM_128MB=y
 CONFIG_SDCARD=y
 CONFIG_AT91_MCI1=y
 CONFIG_JUMP_ADDR="0x23800000"
-CONFIG_DEBUG=y
-- 
1.8.3.1

